mne/time_frequency/tfr.py

```diff
@@ -1511,9 +1513,28 @@ def plot_joint(self, timefreqs=None, picks=None, baseline=None,
             fmin = freq - freq_half_range
             fmax = freq + freq_half_range
 
+            data = tfr.data
+
+            if layout is None:
+                pos = find_layout(tfr.info).pos
+            else:
+                pos = layout.pos
+
+            # merging grads here before rescaling makes ERDs visible
+            if ch_type == 'grad':
+                info = tfr.info
+                picks, new_pos = _pair_grad_sensors(info, find_layout(info))
```

Comment:
Wouldn't dropping 1525 and doing `picks, new_pos = _pair_grad_sensors(tfr.info, find_layout(tfr.info))` be cleaner?